### PR TITLE
UI component updates

### PR DESCRIPTION
**Pull request by jacogr**
- Expand tests for some of the slightly less than straight-forward components (multi-rendering logic) - BlockStatus, ConfirmDialog, TxHash
- Add missing tests for IdentityName/IdentityIcon where missed in bug-fix PRs (e.g. #3950 & children)
- Cleanup some pre-existing PropType warnings in ui/* tests
- Updated strings to be i18n capable in touched components

Not critical for 1.5 merge.

### CODE REVIEW

File: js/src/ui/TxHash/txHash.js [hunk 1]
```diff
@@ -121,10 +124,21 @@ class TxHash extends Component {
...
+            {
+              value === 1
+                ? <FormattedMessage
+                  id='ui.txHash.confirmation'
+                  defaultMessage='{count} confirmation'
+                  values={ { count } } />
+                : <FormattedMessage
+                  id='ui.txHash.confirmations'
+                  defaultMessage='{count} confirmations'
+                  values={ { count } } />
```
Comment (ngotchac) [inline js/src/ui/TxHash/txHash.js]: Isn't there something for formatting variations based on singular/plurals/etc. ? This binary 1 vs. many won't work in some other languages

**Comment by ngotchac** [comment]
One small comment. Looks good otherwise.

File: js/src/ui/TxHash/txHash.js (hunk 1 above)
Comment (derhuerst) [inline js/src/ui/TxHash/txHash.js]: I guess this is a more general problem with our i18n setup.

Comment (jacogr) [inline js/src/ui/TxHash/txHash.js]: There is native support in react-intl, honestly I cannot quite remember how it works, but will take a look and update accordingly.

**Pull request merged**